pkg/kv/bulk/sst_batcher.go

```diff
@@ -123,6 +127,9 @@ type SSTBatcher struct {
 	ms enginepb.MVCCStats
 	// rows written in the current batch.
 	rowCounter storage.RowCounter
+
+	mu  syncutil.Mutex
+	grp ctxgroup.Group
```

Comment:
can we add a comment about what operations the group synchronizes, and when is it reset?

Reply:
I gave it a descriptive name instead. wdyt?